### ISSUE TITLE
URL scheme for external follow links

### ISSUE THREAD

**Issue by gerwitz** [cheeaun/phanpy#488]
Like so many others, I would like simple "follow [account] on the fediverse" links.

With a simple and reliable login flow, phanpy.social could offer a solution in the form of a "follow [account] via Phanpy" link.

I imagine constructing a URL that includes a target account for following. Visiting this URL would lead the user to log into their instance and then confirm following the target account. They could then continue to the usual Phanpy home feed.

**Comment by gerwitz** [comment]
I should acknowledge that if this was in place and became a popular solution for "follow me on the fediverse" links, it could create a ridiculous amount of traffic. Perhaps it should be an optional configuration not enabled on the flagship hosting.

**Comment by cheeaun** [comment]
I'm not really sure what this should do but I do think this should be done on the instance, server or protocol layer - some folks are working on this if I'm not mistaken.

It would be best to treat Phanpy as just an app (like any client-side mobile or desktop apps) instead of a (centralised) proxy site for every instance.

**Comment by gerwitz** [comment]
Yeah, that's why I followed-up. It's not appropriate for Phanpy to become a centralized follow-router, but I *would* like to use it as such for my own instance.

This is a selfish-for-self-hosting feature for sure.
